include/Makefile.am

```diff
@@ -25,13 +25,15 @@ noinst_HEADERS = \
 	tracepoint.h \
 	nccl-headers/net.h \
 	nccl-headers/error.h \
```

Comment:
We are missing some others.

* `nccl-headers/nvidia/types.h`
* `nccl_ofi_memcheck_asan.h`
* `nccl_ofi_memcheck_valgrind.h`

Though admittedly I'm not sure what these definitions do and what needs to be included.

Reply:
Confirmed, passing up asan/valgrind flags to AM does trip on these as well. Fixed in new rev.